depclean-core/pom.xml

```diff
@@ -8,12 +8,12 @@
     <parent>
         <groupId>se.kth.castor</groupId>
         <artifactId>depclean-parent-pom</artifactId>
-        <version>2.0.1-SNAPSHOT</version>
+        <version>2.0.2-SNAPSHOT</version>
```

Comment:
@cesarsotovalero I changed the version, although not sure if this encouraged under the contribution guidelines. Feel free to let me know if this works.